src/test/java/se/sundsvall/billingpreprocessor/api/validation/impl/ValidInvoiceRowsConstraintValidatorTest.java

```diff
@@ -167,24 +168,6 @@ private static AccountInformation createAccountInformation(String costCenter, St
 			.withSubaccount(subAccount);
 	}
 
-	@ParameterizedTest
-	@EnumSource(value = Type.class)
-	void withAccountInformationMissingCounterPart(Type type) {
-		final var billingRecord = BillingRecord.create().withType(type).withInvoice(Invoice.create().withInvoiceRows(List.of(InvoiceRow.create()
-				.withAccountInformation(AccountInformation.create()
-						.withDepartment("Department")
-						.withSubaccount("Subaccount")
-						.withCostCenter("CostCenter")))));
-
-		when(contextMock.buildConstraintViolationWithTemplate(any())).thenReturn(builderMock);
-
-		assertThat(validator.isValid(billingRecord, contextMock)).isFalse();
-
-		verify(contextMock).disableDefaultConstraintViolation();
-		verify(contextMock).buildConstraintViolationWithTemplate("when accountInformation is present costCenter, subaccount, department and counterpart are mandatory");
-		verify(builderMock).addConstraintViolation();
-	}
```

Comment:
Is this not relevant anymore?

Reply:
the parameterized test on line 115 already covers the exact scenario this test was fullfilling (and more).